README.rst

```diff
@@ -21,28 +21,36 @@ Kriging Toolkit for Python
 
 
 
-The code supports two- and three- dimensional ordinary and universal kriging. Standard variogram models (linear, power, spherical, gaussian, exponential) are built in, but custom variogram models can also be used with the code. The kriging methods are separated into four classes. Examples of their uses are shown below. The two-dimensional universal kriging code currently supports regional-linear, point-logarithmic, and external drift terms, while the three-dimensional universal kriging code supports a regional-linear drift term in all three spatial dimensions. Both universal kriging classes also support generic 'specified' and 'functional' drift capabilities. With the 'specified' drift capability, the user may manually specify the values of the drift(s) at each data point and all grid points. With the 'functional' drift capability, the user may provide callable function(s) of the spatial coordinates that define the drift(s). The package includes a module that contains functions that should be useful in working with ASCII grid files (`*.asc`).
+The code supports 2D and 3D ordinary and universal kriging. Standard variogram models (linear, power, spherical, gaussian, exponential) are built in, but custom variogram models can also be used. The 2D universal kriging code currently supports regional-linear, point-logarithmic, and external drift terms, while the 3D universal kriging code supports a regional-linear drift term in all three spatial dimensions. Both universal kriging classes also support generic 'specified' and 'functional' drift capabilities. With the 'specified' drift capability, the user may manually specify the values of the drift(s) at each data point and all grid points. With the 'functional' drift capability, the user may provide callable function(s) of the spatial coordinates that define the drift(s). The package includes a module that contains functions that should be useful in working with ASCII grid files (`*.asc`).
 
-PyKrige is on PyPi, so installation is as simple as typing the following into a command line.
+See the `documentation <http://pykrige.readthedocs.io/en/latest/>`_ for more details.
+
+Installation
+^^^^^^^^^^^^
+
+PyKrige requires Python 2.7 or 3.5+ as well as numpy, scipy and matplotlib. It can be installed from PyPi with,
 
 .. code:: bash
 
     pip install pykrige
 
-To update PyKrige from PyPi, type the following into a command line.
+scikit-learn is an optional dependency needed for parameter tuning and regression kriging.
+
+
+If you use conda, PyKrige can be installed from the `conda-forge` channel with,
 
 .. code:: bash
 
-    pip install --upgrade pykrige
```

Comment:
pip prior to mid 2017 by default has a eager upgreade strategy  https://github.com/pypa/pip/commit/375ed4d3fcf2679dfcd011c3441caa05e5b5c94a  that can break things in the existing packages https://github.com/pypa/pip/issues/304 so I'd rather not put this in the readme.